boards/arm/efm32hg_slstk3400a/doc/efm32hg_slstk3400a.rst

```diff
@@ -0,0 +1,175 @@
+.. _efm32wg_stk3800:
+
+EFM32HG-SLSTK3400A
+###############
+
+Overview
+********
+
+The EFM32 Happy Gecko Starter Kit EFM32WG-STK3800 contains a MCU from the
+EFM32HG family built on ARM® Cortex®-M4F processor with excellent low
+power capabilities.
+
+.. image:: happy-gecko-starter-kit.jpg
+   :width: 375px
+   :align: center
+   :alt: EFM32HG-SLSTK3400A
```

Comment:
If you took the image from Silicon Labs, you should credit them, see https://www.silabs.com/community/general-interest/forum.topic.html/usage_of_board_pictu-ydFZ

I did this there: https://github.com/zephyrproject-rtos/zephyr/pull/6841/commits/3092d6aaf5c864601bae42f8fd0c2abd95521047#diff-60f73521f2a29f145ead3733a9f66c26R13